### PR TITLE
Add aspect-raytio extension

### PR DESCRIPTION
**Pull request by ickas**
## Description

An extension to easily calculate the aspect ratio for different sizes based on width or height, with the possibility to add custom aspect ratios.

## Screencast

aspect-raytio-2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/aspect-raytio/src/action-options.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+              if (handleDeleteItem) {
+                handleDeleteItem(id);
+              }
+              await LocalStorage.removeItem(id);
+            }}
+          />
+        )}
+        <Action
+          icon={{ source: Icon.Important, tintColor: Color.Red }}
+          title="Delete All Custom Raytios"
```
Comment (pernielsentikaer) [inline extensions/aspect-raytio/src/action-options.tsx:76]:
```suggestion
          title="Delete All Custom Raytios"
          style={Action.Style.Destructive}
```
And maybe add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert?q=confirmAlert)

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Use the build-in [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) tool to make the metadata images

- It would be cool to use [arguments](https://developers.raycast.com/information/lifecycle/arguments) for even quicker measurements like this:

image

- use [useForm](https://developers.raycast.com/utilities/react-hooks/useform) in your form instead, that handles validation, errors and states - it's possible to enter text in the fields 🙂




- The ActionPanel items should have a modifier like the your Switch item

image

- Instead of having a `List.Item` showing how to create a new you could make an action on all items with cmd+n and maybe even add a subtitle to the List group. 

image


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by ickas** [comment]
@pernielsentikaer, thanks for all the suggestions; really appreciate it! I applied all of them, and it is ready for a new review!

Using arguments to send the start value directly to the app is 🔥 

The use of `List.Item` on the list to create new custom ratios was more of a UX decision than anything else. I'm afraid the user doesn't see that it has the option for that. But I moved the option to the `ActionPanel`, and I really like how clean it looks now.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**